src/resources/styles.ts

```diff
@@ -44,6 +46,7 @@ export const derivedStyles = {
   "mdc-theme-on-surface": "var(--primary-text-color)",
   "app-header-text-color": "var(--text-primary-color)",
   "app-header-background-color": "var(--primary-color)",
+  "app-header-height": "var(--header-height)",
```

Comment:
Why do we need a separate var for `app-header-height`? Why not use `header-height`

Reply:
done that on purpose independent custom height

Reply:
What would be the use case?

Reply:
@sdotter ?